### PR TITLE
Added eslint-plugin-jsx-a11y to install script

### PR DESCRIPTION
**Pull request by chentsulin**

### CODE REVIEW

File: packages/eslint-config-airbnb/README.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ We export three ESLint configurations for your usage.
 Our default export contains all of our ESLint rules, including EcmaScript 6+
 and React. It requires `eslint` and `eslint-plugin-react`.
```
Comment (lencioni) [inline packages/eslint-config-airbnb/README.md]: Will you please also update this line?

Comment (chentsulin) [inline packages/eslint-config-airbnb/README.md]: Done

File: packages/eslint-config-airbnb/README.md [hunk 2]
```diff
@@ -11,9 +11,9 @@ We export three ESLint configurations for your usage.
 ### eslint-config-airbnb
 
 Our default export contains all of our ESLint rules, including EcmaScript 6+
-and React. It requires `eslint` and `eslint-plugin-react`.
+and React. It requires `eslint`, `eslint-plugin-react` and `eslint-plugin-jsx-a11y`.
```
Comment (ljharb) [inline packages/eslint-config-airbnb/README.md]: hate to be a bother but could we include an Oxford comma here? :-)

Comment (lencioni) [inline packages/eslint-config-airbnb/README.md]: I wonder if we should also linkify these (doesn't need to be part of this PR).

Comment (ljharb) [inline packages/eslint-config-airbnb/README.md]: i don't think it matters, the next line shows them being npm-installed, so you know where the page is

**Comment by ljharb** [comment]
Thank you!

**Pull request merged**